sdk/eventhub/event-hubs/README.md

```diff
@@ -210,8 +210,10 @@ The `EventProcessor` will delegate the processing of events to a [PartitionProce
 that you provide, allowing you to focus on business logic while the processor holds responsibility for managing the underlying consumer
 operations including checkpointing and load balancing.
 
-While load balancing is a feature we will be adding in the next update, you can see how to use the `EventProcessor` in the below
-example, where we use an [InMemoryPartitionManager](https://azure.github.io/azure-sdk-for-js/event-hubs/classes/inmemorypartitionmanager.html) that does checkpointing in memory.
+While load balancing is typically useful when running multiples instances of `EventProcessor` across multiple processes or even hosts,
+you can see it action in the below example, where we use an [InMemoryPartitionManager](https://azure.github.io/azure-sdk-for-js/event-hubs/classes/inmemorypartitionmanager.html)
```

Comment:
```suggestion
You can see it action in the below example, where we use an [InMemoryPartitionManager](https://azure.github.io/azure-sdk-for-js/event-hubs/classes/inmemorypartitionmanager.html)
```

Reply:
Re-word to suggest that one would want to use a durable store, but for now or for getting started you can use the in memory stuff